at-dungnguyen/Week3/2018_07_03_at_dungnguyen3_exercise_3.rb

```diff
@@ -0,0 +1,56 @@
+# Write a Ruby data structure to hold this data.
+teachers = [
+  {
+    name: 'Paul Barry',
+    areas_of_interest: ['Programming', 'Networking', 'Security', 'Open Source', 'Framework']
+  },
+  {
+    name: 'Chris Meudec',
+    areas_of_interest: ['Testing', 'Safety Systems', 'Formal Systems', 'Open Source', 'Programming Languages']
+  },
+  {
+    name: 'Nigel Whytec',
+    areas_of_interest: ['Graphics', 'Imaging', 'Programming', 'Sign Language', 'Trees']
+  },
+  {
+    name: 'Austin Kinsella',
+    areas_of_interest: ['Networks', 'WANs', 'Programming', 'Macintosh', 'Digital Photography']
+  },
+  {
+    name: 'Gerry Moloney',
+    areas_of_interest: ['Placement', 'Employment', 'Emerging Systems', 'Web Development']
+  }
+]
+
+# Show list areas of interest of the first and third lecturers.
+puts teachers.first[:areas_of_interest] if teachers.first
+puts teachers[2][:areas_of_interest] if teachers[2]
+
+# Remove the last lecturer
+teachers.pop
+
+# Change lecturer name from Austin Kinsella to Howard Williams
+teachers.each do |teacher|
+  teacher[:name] = 'Howard Williams' if teacher.value?('Austin Kinsella')
+end
+
+# Prints Lecture Name, who interest in Programming
+teachers.each do |teacher|
+  puts "Teacher: #{teacher[:name]}" if teacher[:areas_of_interest].include?('Programming')
+end
+
+# Change areas of interest Programming to Developing
+teachers.each do |teacher|
+  teacher[:areas_of_interest].map! { |obj| obj == 'Programming' ? 'Developing' : obj }
+end
+
+# Get distince interest
+interests = Array.new
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/aa3e4c4b581783c390e26bc2c405146e3835a479/at-dungnguyen/Week3/2018_07_03_at_dungnguyen3_exercise_3.rb#L48"></span>Style/EmptyLiteral: Use array literal <code>[]</code> instead of <code>Array.new</code>. (https://github.com/rubocop-hq/ruby-style-guide#literal-array-hash)</td>
    </tr>
  </tbody>
</table>